.coderabbit.yaml

```
@@ -0,0 +1,106 @@
+      - "p/nextjs"       # Next.js framework rules
+      - "p/typescript"   # TypeScript language rules
+      - "p/javascript"   # JavaScript language rules
+      - "p/security-audit"  # Security best practices
+      - "p/owasp-top-ten"   # OWASP top 10 vulnerabilities
+  gitleaks:
+    rules:
+      - "default"
+      - "github"
+      - "npm"
+      - "aws"
+      - "google"  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)